tests/sys/psa_crypto_cbor_encoder/Makefile

```diff
@@ -0,0 +1,22 @@
+include ../Makefile.sys_common
+
+USEMODULE += embunit
+
+# These are only needed to get PSA Crypto to build
+USEMODULE += psa_hash
+USEMODULE += psa_hash_sha_256
```

Comment:
Why's that?

Reply:
To run the test for encoding a key pair (or an asymmetric protected key slot), we need a key slot that can hold an ECC P256 public and private key. 
Those key slots only exist and have the right size, when we select the modules `psa_asymmetric` and `psa_asymmetric_ecc_p256r1`, which not only compiles the key slot structure, but also a whole backend implementation. A function like `psa_ecc_p256r1_sign_message` calls `psa_hash_compute`, which does not exist, if we don't compile the hash module.

Reply:
> A function like `psa_ecc_p256r1_sign_message` calls `psa_hash_compute`, which does not exist, if we don't compile the hash module.

But that sounds like `psa_asymmetric_ecc_p256r1` should depend on `psa_hash_sha_256`, so users don't need to figure out these dependency issues by themselves?

Reply:
A user could use a different hash (see https://doc.riot-os.org/crypto__values_8h.html#a7e3ce9f514a227d5ba5d8318870452e3), so they still have to choose it themselves. Our example just uses this one.